src/pages/help-centre/_search-results.tsx

```diff
@@ -1,11 +1,25 @@
 import React from 'react'
-import PropTypes from 'prop-types'
 import styled from 'styled-components'
 import { convertToHash } from './_utility'
 import { Text, Header, LocalizedLinkText } from 'components/elements'
 import { localize } from 'components/localization'
 import device from 'themes/device'
 
+type TopicsType = {
+    category: string
+    label: string
+    title: string | React.ReactElement
+}[]
```

Comment:
Here as well no need to make types array. Instead you can use it as an array when typing a variable